MetaMask/metamask-mobile
# test: ERC1155 set approval for all e2e

## **Description**

This PR is within the scope of the Quality Quest.

We're including an e2e test case to check the set approval for all for ERC1155 in test dApp.

**Test steps:**
Given I am on the test dapp
When I tap on the Set Approval for All button under the ERC 1155 section
Then the transaction bottom sheet should appear
When I submit the transaction
Then the transaction should appear in the transaction history

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Screenshots/Recordings**

### **Before**

### **After**

## **Pre-merge author checklist**

- [✓] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [✓] I've completed the PR template to the best of my ability
- [✓] I’ve included tests if applicable
- [✓] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [✓] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**
